src/compiler/utilities.ts

```diff
@@ -2513,14 +2513,15 @@ namespace ts {
     }
 
     export function getEffectiveBaseTypeNode(node: ClassLikeDeclaration | InterfaceDeclaration) {
-        if (isInJSFile(node)) {
+        const baseType = getClassExtendsHeritageElement(node);
+        if (baseType && isInJSFile(node)) {

```

Comment:
@DanielRosenwasser @sandersn should it be error to have augments clause and not have extends clause?

Reply:
I think it would be too noisy. Every time we add errors like this they add noise to erroneous jsdoc -- ones where people weren't actually using the jsdoc and didn't want to.